ts/services/notifications.ts

```diff
@@ -393,10 +393,12 @@ class NotificationService extends EventEmitter {
           notificationMessage = i18n('icu:newMessage');
         } else if (userSetting === NotificationSetting.NameOnly) {
           if (reaction) {
-            notificationMessage = i18n('icu:notificationReaction', {
-              sender: senderTitle,
-              emoji: reaction.emoji,
-            });
+            notificationMessage = NotificationSetting.NameOnly
+              ? i18n('icu:notificationReactionNoEmoji', { sender: senderTitle })
+              : i18n('icu:notificationReaction', {
+                  sender: senderTitle,
+                  emoji: reaction.emoji,
+                });
           } else {
```

Comment:
Hey there - thanks for doing this. Instead of introducing a new string, let's match Android, which just says "New Message." We can get rid of the `if (reaction)` entirely!

Reply:
Done.